pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -394,6 +406,18 @@ func (c *Cluster) ConfigureArbiter() error {
 	return nil
 }
 
+func (c *Cluster) builtinMgrPoolExists(checkOSDPods bool) bool {
```

Comment:
```suggestion
func (c *Cluster) builtinMgrPoolExists(checkOSDPods bool) bool {
```
`checkOSDPods` argument is not used in the function.